ui/src/app/pages/data/data-page.tsx

```diff
@@ -114,7 +115,7 @@ export const DataPage = withCurrentWorkspace()(class extends React.Component<
         isLoading: true
       });
       const [cohorts, cohortReviews, conceptSets, dataSets] = await Promise.all([
-        cohortsApi().getCohortsInWorkspace(namespace, id),
+        fetchWithGlobalErrorHandler(() => cohortsApi().getCohortsInWorkspace(namespace, id)),
```

Comment:
This is an example of how you would manually call it. If you wanted error handling that was not done via the global handler, you would add it after the api call within the `fetchWithGlobalErrorHandler` method.